src/libponyc/type/cap.c

```diff
@@ -912,49 +970,145 @@ bool cap_immutable_or_opaque(token_id cap)
   return false;
 }
 
-bool cap_safetomove(token_id into, token_id cap, direction direction)
+
+
+static ast_t* modified_cap_single(ast_t* type, cap_mutation* mutation)
 {
```

Comment:
Related to the above, I believe that the Git diffing algorithm has decided that this was derived from the cap safe_to_move above, so I would recommend _not_ attempting to make sense of the diff and just looking at the raw code

Reply:
In any case, this is only used right now for the unisolation bit. More code can be made to use this, but that starts bringing up the scope (as there's also bugs caused by the way that some of those are implemented today)